Corrections to the English translation, additions and corrections to the Russian translation, typos corrections

maximus105 opened the pull request:
Summary
=======
1. Added translation for untranslated words in the menu into Russian.
2. Added translation for the word "Frequency" for all language files.
3. Corrections of errors in translation.
4. Corrected the word "Speed" for processors to the word "Frequency" for all language files. At the same time, the word "Speed" for HDD and CD-ROM remains as before.
5. Added the word "3Dfx" for Voodoo Graphics.
6. Word corrections for IBM 8514/A Graphics and XGA Graphics.
7. Word corrections for OpenGL 3.0 Renderer options.
8. Word corrections for Joystick Configuration.
9. Corrected the word "3dfx" to "3Dfx".
10. Added the word "3Dfx" for Voodoo video cards.
11. Corrected the word "max. speed" to "Max. Speed" for HDD.

Checklist
=========
* [ ] Closes #xxx
* [ ] I have discussed this with core contributors already
* [ ] This pull request requires changes to the ROM set
  * [ ] I have opened a roms pull request - https://github.com/86Box/roms/pull/changeme/

References
==========
_Provide links to datasheets or other documentation that helped you implement this pull request._

binarymaster commented on src/qt/languages/ru-RU.po:
```diff
@@ -25,11 +25,17 @@ msgstr "&Выход..."
-msgid "Hide &toolbar"
-msgstr "С&крыть панель инструментов"
+msgid "Show status icons in fullscreen"
+msgstr "Показывать значки статуса в полноэкранном режиме"
```
```suggestion
msgstr "Показывать значки состояния в полноэкранном режиме"
```
(consistency with `msgid "&Hide status bar"`)

binarymaster commented on src/qt/languages/ru-RU.po:
```diff
@@ -50,13 +56,37 @@ msgid "SDL (&OpenGL)"
 msgstr "&VNC"
 
+msgid "Renderer options..."
+msgstr "Параметры рендеринга OpenGL 3.0..."
```
```suggestion
msgstr "Параметры рендеринга..."
```
Probably a copypaste, check original msgid.

binarymaster commented on src/qt/languages/ru-RU.po:
```diff
@@ -127,6 +157,9 @@ msgstr "&Целочисленное масштабирование"
 msgstr "4:&3 Целочисленное масштабирование"
 
+msgid "Apply fullscreen stretch mode when maximized"
+msgstr "Применить полноэкранный режим растяжения при максимальном увеличении"
```
```suggestion
msgstr "Применить полноэкранный режим растяжения при разворачивании окна"
```
"maximized" is a term for the window state.

binarymaster commented on src/qt/languages/ru-RU.po:
```diff
@@ -232,6 +268,9 @@ msgstr "&Перемотка в конец"
 msgstr "И&звлечь"
 
+msgid "Eject %s"
+msgstr "И&звлечь"
```
```suggestion
msgstr "Извлечь %s"
```
Check original msgid.

maximus105 replied:
This line was added for Floppy. To apply the translation for the word "Eject" (Under the line "Export to 86F").

binarymaster reviewed (commented):
Reviewed ru-RU.po

OBattler commented:
Please correct the relevant strings in the other language files as well, or the other translations are going to be broken.

maximus105 commented:
> Please correct the relevant strings in the other language files as well, or the other translations are going to be broken.

Corrected the word "Speed" for processors to the word "Frequency" for all language files, added a translation to the word "Frequency" for all language files
